pom.xml

```diff
@@ -32,6 +32,7 @@
     <!-- TODO: Ongoing work on https://github.com/MarkEWaite/git-plugin/tree/master-findbugs-fixes. -->
     <findbugs.failOnError>false</findbugs.failOnError>
     <workflow.version>1.14.2</workflow.version>
+    <scm-api-plugin.version>2.0.8-20170228.204551-2</scm-api-plugin.version> <!-- TODO https://github.com/jenkinsci/scm-api-plugin/pull/32 -->
```

Comment:
The last time I tried to perform interactive tests with this type of dependency expressed in the pom file, it failed to load.  The report was that the dependent version of the plugin could not be found (since that long version string is not the version string of the build).

Is there a plan to fix that behavior in a future Jenkins release?

I assume that interactive testing of all versions after this is merged will need to mutate the version of the scm-api-plugin from the explicit version back the 2.0.8-SNAPSHOT (until scm-api 2.0.8 is released).  Is that a save assumption?